SWIPs/swip-multiple_payment_processing_support.md

```diff
@@ -13,11 +13,11 @@ created: 2019-07-22
 
 ## Simple Summary
 <!--"If you can't explain it simply, you don't understand it well enough." Provide a simplified and layman-accessible explanation of the SWIP.-->
```

Comment:
I can not leave comments to things which have not changed, so here are some further comments:

* I don't know if it is because this PR has been opened before, but the code still shows the old code for message handling (`handleMsg`), this should be updated
* The Interface methods are most probably going to be public, so use Uppercase
* Shouldn't the `PaymentParams` also include a `Custom` field or something, for the extra method-specific data?
* The "Rationale" section lists alternative solutions, but I think there is a closing sentence missing, explaining why none of the above were chosen and why the proposed solution is the one presented

This sWIP probably ties in with the **capabilities** SWIP which was going prepared by another track, which is about initial advertisement/negotiations between nodes about their respective capabilities. Nevertheless I don't think that SWIP is much advanced, but I thought it's good I mention it here. If I find a reference, we may want to mention it here.